src/example.wordswipe-test.config.js

```diff
@@ -5,14 +5,21 @@ export default {
     databaseURL: "https://wordswipe-test-default-rtdb.firebaseio.com",
     projectId: "wordswipe-test",
     storageBucket: "wordswipe-test.appspot.com",
+    messagingSenderId: "934860809801",
+    appId: "1:934860809801:web:b59be557ac5a62105a4198",
+    measurementId: "G-0MCHLFQVZ9",
   },
   home: {
     title: "Rapid Online Assessment of Reading",
     tagline: "Are the words real or fake?",
   },
-  needsTutorial: true,
+  // shufflingMethod: "random",
+  shufflingMethod: "leastSeen",
+  needsTutorial: false,
   manifestUrl: "",
-  widgetType: "TimedWordSwipe",
+  widgetType: "WordSwipe",
+  mode: "Assessment", // "Data Annotation"
```

Comment:
Some comments might be good for new config values explaining the possible values they can take.